docs/getting-started/guides/how-boost-works.md

```diff
@@ -76,7 +82,7 @@ This is a “set-and-forget” vault where your CRV tokens grow exponentially, h
 
 Now you might be wondering how one would extract any gains made from your CRV tokens in the vault, when as mentioned earlier, any CRV deposited into either the yveCRV or the yvBOOST are locked. While you cannot withdraw from the yveCRV vault, you can actually swap both of these vault tokens on Sushiswap. This is because Yearn and its users provide liquidity on Sushiswap to allow swapping of your yveCRV and yvBOOST tokens for ETH (or anything, really).
 
-#### A little alpha.
+#### A little alpha
 
 Yearn buys yvBOOST from the market, unwraps it into yveCRV, and donates that yveCRV into the yvBOOST vault, increasing the underlying value of yvBOOST.
```

Comment:
is this still true?

Reply:
I thought it was close enough to how I assume it works now: In the yvyCRV vault, it sells stables earned for yCRV to be deposited back into the pool.

I have the new info disclaimer at the top of the page saying these are deprecated and to look at the current yCRV docs.